apps/onboarding-api/onboarding_api/app.py

```diff
@@ -0,0 +1,135 @@
+from flask import Flask, jsonify, request, g, redirect, url_for
+import sqlite3
+import os
+
+app = Flask(__name__)
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+DATABASE_PATH = os.path.join(BASE_DIR, "tasks.db")
+INIT_SCHEMA = os.path.join(BASE_DIR, "tasks_schema.sql")
+
+
+def get_db():
+    db = getattr(g, "_database", None)
+    if db is None:
+        db = g._database = sqlite3.connect(DATABASE_PATH)
+    # queries return rows instead of unnamed tuples
+    db.row_factory = sqlite3.Row
+    return db
+
+
+@app.teardown_appcontext
+def close_connection(exception):
+    db = getattr(g, "_database", None)
+    if db is not None:
+        db.close()
+
+
+def init_db():
+    with app.app_context():
+        db = get_db()
+        with app.open_resource(INIT_SCHEMA, mode="r") as f:
+            db.cursor().executescript(f.read())
+        db.commit()
+
+
+@app.cli.command("initdb")
+def initdb_command():
+    init_db()
+    print("Initialized the sqlite3 database")
+
+
+def query_db(query, args=(), one=False):
+    cur = get_db().execute(query, args)
+    rv = cur.fetchall()
+    cur.close()
+    return (rv[0] if rv else None) if one else rv
+
+
+def change_db(query, args=()):
+    db = get_db()
+    cur = db.execute(query, args)
+    db.commit()
+    db.close()
+    return cur.lastrowid
+
+
+@app.get("/health")
+def health():
+    return jsonify({"status": "ALIVE"})
+
+
+@app.route("/api/tasks/create", methods=["POST"])
```

Comment:
should remove the `/create` at the end because the method `POST` already indicates its an operation to create something